src/io/memory_block_io.h

```diff
@@ -109,14 +123,18 @@ class MemoryBlockIO : public BasicIO<MemoryBlockIO> {
     Allocator* const allocator_{nullptr};
 
     static const uint64_t DEFAULT_BLOCK_SIZE = 128 * 1024 * 1024;  // 128MB
+
+    uint64_t block_bit_ = 27;
```

Comment:
block_bits = 27 corresponds to a DEFAULT_BLOCK_SIZE of 128MB. Should we also define a corresponding DEFAULT_BLOCK_BITS = 27 here ?

Reply:
done